src/main/java/com/example/demo/service/MatchService.java

```diff
@@ -0,0 +1,70 @@
+package com.example.demo.service;
+
+import com.example.demo.models.Match;
+import com.example.demo.models.Team;
+import org.springframework.stereotype.Component;
+
+import java.util.*;
+@Component
+public class MatchService {
+    private List<Match> upcomingMatches = new ArrayList<>();
+    private List<Match> playedMatches = new ArrayList<>();
+
+     int a = 2;
+
+    private final TeamService teamService;
+
+    public MatchService(TeamService teamService) {
+        this.teamService = teamService;
+    }
+
+
+    public  Match playMatch() {
+        Match match = upcomingMatches.get(0);
```

Comment:
Try to not use numbers in methods. Why we have 0 here? There is better way - create static final variable as const value and use it here.